ambry-server/src/integration-test/java/com.github.ambry.server/ServerHardDeleteTest.java

```diff
@@ -293,7 +294,7 @@ public void endToEndTestHardDeletes() throws Exception {
     notificationSystem.awaitBlobDeletions(blobIdList.get(4).getID());
 
     time.sleep(TimeUnit.DAYS.toMillis(1));
-    ensureCleanupTokenCatchesUp(chosenPartition.getReplicaIds().get(0).getReplicaPath(), mockClusterMap, 198520);
+    ensureCleanupTokenCatchesUp(chosenPartition.getReplicaIds().get(0).getReplicaPath(), mockClusterMap, 198902);
```

Comment:
What is the math for the increase of 382 bytes? Can you figure that out?

Reply:
Will sync up with you offline. I traced the putRecord Size and DeleteRecordSize and came up with this. But would like to get it done programmatically. Wanted to understand why we can't do so.

Reply:
Here is the math for the delta: every header is increased by 4 bytes. Every BlobProperties have been increased by 1 byte. If encrypted, we add 114 bytes (if encrptionKey is of size 100) to the total. 

6 puts (3 encrypted)
1 delete

=> 7*4 (for header) + 1*6(for blob props) + 114*3 (for encryption record) = 376 is the delta